packages/sommelier.rb

```diff
@@ -204,6 +259,13 @@ def self.build
             pkill -F #{CREW_PREFIX}/var/run/sommelier-xwayland.pid &>/dev/null
             DISPLAY="${DISPLAY:0:3}"
 
+            xdg_shell_v6=''
+
+            # check if exo support xdg_wm_base
+            if [[ -z "$(wayland-info -i xdg_wm_base)" ]]; then
```

Comment:
So my machine works fine with this logic reversed, that is, without the additional flag.